src/Xamarin.Android.Cecil/Xamarin.Android.Cecil.targets

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <CecilDirectory>$(MSBuildThisFileDirectory)\..\..\external\cecil</CecilDirectory>
+    <CecilPreparedFlag>prepared.flag</CecilPreparedFlag>
+    <OutputPath Condition="'$(OutputPath)' == ''">bin\$(Configuration)</OutputPath>
+    <CecilOutputPath>$([System.IO.Path]::GetFullPath ('$(OutputPath)'))</CecilOutputPath>
+  </PropertyGroup>
+  <Target Name="PrepareCecil"
+      Inputs="$(CecilDirectory)\Mono.Cecil.sln"
+      Outputs="$(CecilPreparedFlag)">
+    <Exec Command="cd $(CecilDirectory); ln -sf $(MSBuildThisFileDirectory)\AssemblyInfo.cs Mono.Cecil.AssemblyInfo.cs" />
```

Comment:
A medium-term (short-term?) goal for the Java.Interop and xamarin-android repos is to "support" building the projects on Windows.

We shouldn't deliberately introduce anything that would prevent that from working. It might not work for other reasons -- e.g. mono builds on Windows are something I'm trying to avoid in xamarin-android, hence the `bundle` generation & caching -- but we shouldn't deliberately introduce new constructs which we *know* won't work on Windows.

`ln -s` is such a construct. Perhaps it should use the `<Copy/>` task?

Unfortunately, `patch` is also such a construct. I'm not sure how best to handle that; I guess we should keep it for now.